### ISSUE TITLE
Support vim's j and k keybindings

### ISSUE THREAD

**Issue by ad-si** [2mol/pboy#47]
I'm actually quite surprised if a TUI doesn't support j for down and k for up 😅

**Comment by m040601** [comment]
+1 one more vote for this.

**Comment by 2mol** [comment]
yeah, cool idea! I'll see to it if I have some free time, I'm pretty sure it's just adding a line here: https://github.com/2mol/pboy/blob/a79b07db6228eb3ce234d59f5c2c8b10c5f28ff3/src/UI.hs#L257

Brick probably has some easy way to express the up/down action on a list. Feel free to PR if you happen to be faster than me!

Cheers
